lib/responses.test.js

```diff
@@ -410,4 +410,37 @@ describe(_.startCase(filename), function () {
       fn()(req, res);
     });
   });
+
+  describe('checkArchivedToPublish', function () {
+    const fn = lib[this.title];
+
+    it('throws an error if checkProps returns true', function () {
+      let req = {
+        body: {archived: true},
```

Comment:
Please mind spacing when declaring objects
```suggestion
        body: { archived: true },
```